docs/getting-started/shipping-some-logs.md

```diff
@@ -2,56 +2,142 @@
+## Shipper
 
-The configuration file will attempt to forward `/var/log/system.log`, `/var/log/syslog`, and anything from `stdin`. If
-those inputs are missing, you'll see a warning which you can ignore. On Mac OS X, you could forward
-`/var/log/system.log` (by default) and `/var/log/authd.log` (via `stdin`) by running...
+We call the piece of software that does the shipping of the log file a shipper.  This could be a general piece of software (such as [Logstash](http://www.elasticsearch.org/guide/en/logstash/current/index.html) or [nxlog](http://nxlog-ce.sourceforge.net/)) configured to tail a log file and send its contents to the Logsearch cluster, or a specialised lightweight piece of software that only does log tailing (like [Logstash-forwarder](https://github.com/elasticsearch/logstash-forwarder)).
 
-    $ tail -f /var/log/authd.log | ./logstash-forwarder -config logstash-forwarder.json
+In this guide we will be configuring the Logstash installed by default in the Logsearch Workspace to be the shipper.
 
-You can generate some events by opening Terminal windows or running some commands with `sudo`. After a few messages have
-been sent, you can visualize them by opening Kibana on the `api/0` node:
+## Ingestor
 
-    http://10.244.2.2/_plugin/kibana/#/dashboard/file/default.json
+A Logsearch cluster has one (or many) ingestors, which listen for logs at a specific IP address / TCP/UDP port combo.
 
-You should see a few recent bars on the right side of the top chart, as well as the raw log messages in the table below.
-Since the parser doesn't really know what the format of these messages is, there's only a single, `@message` field with
-the log message.
+We configure which ingestors to run, and where they should listen in the Logsearch cluster manifest, eg:
 
+    ...snip...
+    jobs:
+    - name: ingestor
+      release: logsearch
+      templates:
+      - name: ingestor_syslog
+      instances: 1
+      resource_pool: warden
+      networks:
+      - name: warden
+        static_ips:
+        - 10.244.10.6
+    ...snip...
+    properties: 
+      logstash_ingestor:
+        syslog:
+          port: 514
+    ...snip...
 
----
+The Logsearch cluster we have just deployed is configured to listen on 10.244.2.10:514 for syslog log events.
+
+## Transport protocol
+
+Log events are encoded in a transport protocol when sent `over the wire`. The transport protocol is often encrypted using SSL. 
+
+In this guide we will be using the standard unencrypted [Syslog RFC5242](https://tools.ietf.org/html/rfc5424) protocol.
+
+## Log parsing
+
+A logsearch cluster needs to be deployed with a set of Log parsing rules that configure how the raw log data should be parsed and enriched before being indexed in Elasticsearch as key=>value pairs.
+
+In this guide the Logsearch cluster has been deployed with the [Logsearch for Weblogs Addon](https://github.com/logsearch/logsearch-for-weblogs), so it knows how to parse standard Nginx access logs.
```

Comment:
This does not match the repository name, so either one needs to be adjusted: logsearch-for-weblogs vs. logsearch-for-websites